frontend/server/src/Controllers/Contest.php

```diff
@@ -206,21 +206,25 @@ public static function apiAdminList(\OmegaUp\Request $r) {
     /**
      * Callback to get contests list, depending on a given method
      * @param \OmegaUp\Request $r
-     * @param $callback_user_function
-     * @return array
+     * @param $callbackUserFunction
```

Comment:
```suggestion
     * @param callable(int, int, int):array{.......} $callbackUserFunction
```

hay que pasarle esto como una función anónima en vez de una cadena.

Reply:
Listo